File: java/src/main/java/com/fengtian/qianer/atcoder/abc/abc162/ABC162_B.java
package com.fengtian.qianer.atcoder.abc.abc162;

import java.util.Scanner;

public class ABC162_B {

    public static void main(String[] args) {
        final Scanner sc = new Scanner(System.in);

        final int n = sc.nextInt();

        long ans = 0;

        for (int i = 0; i <= n; i++) {
            if (i % 3 != 0 && i % 5 != 0) ans += i;
        }

        System.out.println(ans);
    }
}

File: kotlin/src/abc/abc162/ABC162_B.kt
package abc.abc162

import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    val n = sc.nextInt()

    val ans = (1..n).filter { it % 3 != 0 && it % 5 != 0 }.sumByDouble(Int::toDouble).toLong()
    println(ans)
}